pkg/services/alerting/rule.go

```diff
@@ -140,9 +140,10 @@ func NewRuleFromDBAlert(ruleDef *models.Alert) (*Rule, error) {
 		if id, err := jsonModel.Get("id").Int64(); err == nil {
 			uid, err := translateNotificationIDToUID(id, ruleDef.OrgId)
 			if err != nil {
-				return nil, ValidationError{Reason: "Unable to translate notification id to uid, " + err.Error(), DashboardID: model.DashboardID, AlertID: model.ID, PanelID: model.PanelID}
+				logger.Debug("Unable to translate notification id to uid", "error", err.Error(), "DashboardID", model.DashboardID, "AlertID", model.ID, "PanelID", model.PanelID)
```

Comment:
Suggest error level on log message or warning.

```suggestion
				logger.Error("Unable to translate notification id to uid", "error", err.Error(), "dashboardId", model.DashboardID, "alertId", model.ID, "panelId", model.PanelID, "notificationId", id)
```